sql/database.sql

```diff
@@ -2649,7 +2650,7 @@ CREATE TABLE `lang_languages` (
 -- Dumping data for table `lang_languages`
 -- 
 
-INSERT INTO `lang_languages` VALUES (1, 'en', 'English');
+INSERT INTO `lang_languages` VALUES (1, 'en', 'English', 0);
 
 -- --------------------------------------------------------
```

Comment:
nice job on your first openemr commit,

I just added support for the rtl setting in the translation spreadsheet and the translation pipeline on the openemr master branch. So, after address above issues for this code, then instead issue a pull request on the openemr repository(and work from the master branch on that repository):
 https://github.com/openemr/openemr

thanks,
-brady